src/features/integration/util/MapToDto.ts

```diff
@@ -0,0 +1,260 @@
+import IFormData from "../types/Form/FormData";
+import {IConfigurationDto} from "../types/ConfigurationDto";
+import {VALUE_BUILDER_STRATEGY} from "../types/ValueBuilderStrategy.enum";
+
+export function mapToDto(data: IFormData): IConfigurationDto {
```

Comment:
What are you mapping to? Name the method according to what it is. `mapToDto` is to generic.

Reply:
changing to "toIntegrationConfiguration"